lnwallet/channel.go

```diff
@@ -5996,16 +5996,49 @@ func (lc *LightningChannel) availableCommitmentBalance(view *htlcView) (
 	)
 	feePerKw := filteredView.feePerKw
 
+	// Set the dustlimit based on who's commitment we are looking at.
+	dustlimit := lnwire.NewMSatFromSatoshis(
```

Comment:
This always uses the dust value of the local channel rather than selecting the proper value based on the commitment we're looking at.

Reply:
Changes in later commit